elastic/kibana
# [ResponseOps][Connectors] Allow to use POST method for get case information in case management webhook

## Summary

Resolves https://github.com/elastic/kibana/issues/178074

This PR allows to use POST method and JSON payload for body for get case information in case management webhook.

<img width="1279" alt="Screenshot 2024-10-24 at 15 02 33" src="https://github.com/user-attachments/assets/aaabc5b8-cd9e-46d6-aab8-85a27731bd88">

### How to test
- Create a case management webhook connector [as per documentation](https://www.elastic.co/guide/en/kibana/master/cases-webhook-action-type.html)
- Use any public API which supports GET and POST methods 
- Use POST method for Get case information
- Verify that it validates the URL and JSON payload correctly 
- Test the connector using Test tab
- Create a rule with action using case management webhook connector
- Verify alerts are generated and action executed without any errors

### Checklist

Delete any items that are not applicable to this PR.
